pages/1.7/administration/faq/index.md

```diff
@@ -17,6 +17,8 @@ We recommend starting with a fresh cluster to ensure all defaults are set to exp
 ## Q. What are the OS requirements of DC/OS?
 See the [system requirements](../installing/ent/custom/system-requirements/).
 
+[](http://yuml.me/diagram/scruffy/class/[Administrator]-^[Person],%20[Person]-^[Party])
```

Comment:
hmm where is this coming from? Is this a rebased commit ? I wonder if it makes sense as it is an Link without any shown character

Reply:
I  was trying files from staging, on the image checker. And I think these two went in by mistake. I will correct that